src/main/java/duke/Task.java

```diff
@@ -0,0 +1,76 @@
+package duke;
+
+/**
+ * The task object.
+ */
+public class Task {
```

Comment:
I would say `Task` can be made an abstract class to avoid instantiation of a Task that is not a ToDo, a Deadline or an Event.